website/content/en/docs/component-guides/kserve/access-aws-service-from-kserve.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "Access AWS Service from KServe"
+description = "Configuration for accessing AWS services for inference services such as pulling images from private ECR and downloading models from S3 bucket."
+weight = 10
++++
+
+## Access AWS Service from Kserve with IAM Roles for ServiceAccount（IRSA）
+1. Export env values:
+    ```bash
+    export CLUSTER_NAME="<>"
+    export CLUSTER_REGION="<>"
+    export PROFILE_NAMESPACE="<>"
+    export SERVICE_ACCOUNT_NAME="<>"
+    ```
+
+
+1. Create Service Account with IAM Role using [IRSA](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html). The following command attaches both `AmazonEC2ContainerRegistryReadOnly` and `AmazonS3ReadOnlyAccess` IAM policies:
+    ```
+    eksctl create iamserviceaccount --name ${SERVICE_ACCOUNT_NAME} --namespace ${PROFILE_NAMESPACE} --cluster ${CLUSTER_NAME} --region ${CLUSTER_REGION} --attach-policy-arn=arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly --attach-policy-arn=arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess  --override-existing-serviceaccounts --approve
+    ```
+     > NOTE: You can use ECR (`AmazonEC2ContainerRegistryReadOnly`) and S3 (`AmazonS3ReadOnlyAccess`) ReadOnly managed policies. We recommend creating fine grained policy for production usecase. 
+
+### Deploy models from S3 Bucket 
+1. Create Secret with empty AWS Credential:
+    ```yaml
```

Comment:
we can make it easier for user by doing something like,
```
cat <<EOF > secret.yaml
apiVersion: v1
    kind: Secret
    metadata:
      name: aws-secret
      namespace: ${PROFILE_NAMESPACE}
      annotations:
        serving.kserve.io/s3-endpoint: s3.amazonaws.com
        serving.kserve.io/s3-usehttps: "1"
        serving.kserve.io/s3-region: ${CLUSTER_REGION}
    type: Opaque
    data:
      AWS_ACCESS_KEY_ID: ""
      AWS_SECRET_ACCESS_KEY: ""
EOF

kubectl apply -f secret.yaml
```

this aligns with next step where you have runnable command for patching the service account